components/brave_news/browser/brave_news_controller.cc

```diff
@@ -767,6 +768,20 @@ void BraveNewsController::ConditionallyStartOrStopTimer() {
           }
         },
         base::Unretained(this)));
```

Comment:
Out of curiosity, why are we passing unretained(this) unlike the new code? If it's mere legacy, can we use WeakPtr for this too?

Reply:
Yeah, it's just legacy - I'll update.